runtime/src/storage/mkvs/tree/lookup.rs

```diff
@@ -118,6 +142,10 @@ impl Tree {
             NodeKind::Leaf => {
                 // Reached a leaf node, check if key matches.
                 let node_ref = node_ref.unwrap();
+
+                // Note: No need to include in the proof builder as the leaf node is always included
+                // with the internal node itself.
```

Comment:
Hm, what is this comment about? In the Go version this would be done when recursing `_get` above for internal node to not pass the proof builder.